.github/workflows/ci-workflow.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.14.7
+          go-version: 1.14.10
```

Comment:
For some reason, when using go version 1.14.4, the step _TestNon3GPP_ gets an error. Using go version 1.14.10 fixed the error.